docs/sidebars.js

```diff
@@ -319,6 +322,18 @@ const sidebars = {
       ],
     },
 
+    {
+      label: "Limitations",
```

Comment:
Why is this not just giving the path, but defining the entire doc etc? Similar for below. If there is only one file, no need to add the extra boilerplate.